test/docs/__snapshots__/get-docs-yaml.api-test.ts.snap

```diff
@@ -2724,11 +2724,10 @@ components:
       properties:
         alternateIdentifier:
           type: string
-          description: The UKEF ID for the party. Should contain just digits.
+          description: The UKEF ID for the party.
           minLength: 8
           maxLength: 8
-          example: '00291013'
-          pattern: /^d{8}$/
```

Comment:
Did we drop this pattern deliberately?

Reply:
No, that was a mistake, good spot, have restored